src/app/homepage/pages/recipes/terminus/terminus.component.html

```diff
@@ -0,0 +1,95 @@
+    checks for any HTTP application.
+    The module <a href="https://github.com/nestjs/terminus">@nestjs/terminus</a> integrates the terminus
+    library with the NestJS ecosystem. To get started with <code>@nestjs/terminus</code> we need to install
+    the required dependencies.
+  </p>
+  <pre><code class="language-typescript">{{ dependencies }}</code></pre>
+  <h4>Setting up a Health Check</h4>
+  <p>
+    A health check represents a summary of <strong>health indicators</strong>. A health indicator
+    executes a check of a service, whether it is up or down. A health check is positive, if all
+    the assigned health indicators are up and running. Because a lot of applications will need similar
+    health indicators, <a href="https://github.com/nestjs/terminus">@nestjs/terminus</a> provides a set
+    of already predefined health indicators, such as:
+  </p>
+  <ul>
+    <li>
+      DatabaseHealthIndicator
+    </li>
+    <li>
+      DNSHealthIndicator
+    </li>
+  </ul>
+  <p>
+    The first step to get started with our first health check,
+    is to setup a service which will associate health indicators
+    to an endpoint.
+  </p>
+  <span class="filename">terminus-options.service.ts</span>
+  <pre><code class="language-typescript">{{ terminusOptionsService }}</code></pre>
+  <p>
+    Once we have set up our <code>HealthService</code>, we can import the <code>TerminusModule</code> into the root
+    <code>ApplicationModule</code>.
+    The <code>HealthService</code> will all provide the settings, which will be used by the <code>TerminusModule</code>
+  </p>
+  <span class="filename">app.module.ts</span>
+  <pre><code class="language-typescript">{{ appModule }}</code></pre>
+  <blockquote class="info">
+    <strong>Hint</strong>
+    If done correctly Nest will expose the defined health check(s), which are reachable through a GET request to the
+    defined route. For example <code>curl -X GET 'http://localhost:3000/health'</code>
+  </blockquote>
+  <h4>Creating a custom Health Indicator</h4>
+  <p>
+    In some cases the predefined health indicators provided by <code>@nestjs/terminus</code> do not cover all of your
+    health check requirements. In this case you can set up a custom health indicator according to your needs.
+  </p>
+  <p>
+    Lets get started by creating a service which will represent our custom health indicator. To get a basic
+    understanding
+    how a health indicator is structured, we will create an example <code>DogHealthIndicator</code>. This health
+    indicator should have the state "up", if every <code>Dog</code> object has the type <code>goodboy</code>,
+    otherwise it will throw an error, which then the health indicator will be seen as "down".
+  </p>
+  <blockquote class="error">
+    <strong>Hint</strong>
+    In a real world application it is seen as best practice to implement the logic
+    of fetching the required data in a separate service. The static array of dogs inside
```

Comment:
Can you reword this sentence? I am not sure what you are trying to say here. I guess you mean splitting the code into a nest module and service instead having it altogether in one file?